boards/DCM/Inc/App/App_Imu.h

```diff
@@ -0,0 +1,44 @@
+#pragma once
+#include <stdint.h>
+#include <stdbool.h>
+
+struct Imu;
+
+/** Creates Imu struct used for reading data from sensor
+ * @param update_data - pointer to function which updates data and returns true
+ * if update successful
+ * @param get_acceleration_x - pointer to function which returns x acceleration in m/s^2
```

Comment:
nit; Some code from the past may be misleading but for now `-` here isn't necessary.

Reply:
ok, removed it